sopel_modules/rat_board.py

```diff
@@ -1707,8 +1713,11 @@ def func_delete(bot, trigger, id):
             return
         bot.say('Deleted case with id ' + str(id) + ' - THIS IS NOT REVERTIBLE!')
     else:
-        result = callapi(bot, 'GET', uri='/rescues?data={"markedForDeletion":{"marked":true}}',
-                         triggernick=str(trigger.nick))
+        try:
+            result = callapi(bot, 'GET', uri='/rescues?data={"markedForDeletion":{"marked":true}}', triggernick=str(trigger.nick))
+        except ratlib.api.http.APIError:
+            bot.reply('Couldn\'t find cases marked for deletion due to API error.')
```

Comment:
```suggestion
            bot.reply("Couldn't find cases marked for deletion due to API error.")
```

Reply:
copypasta from other messages that already existed in the codebase. I don't care.

Reply:
/shrug